### ISSUE TITLE
Enable easy first five minute CX with docker images

### ISSUE THREAD

**Issue by amcp** [JanusGraph/janusgraph#140]
Right now, it is difficult for people to get started with JanusGraph as we have not made an official release. To hold over the community until that happens, lets make it easy for people to get up and running on JanusGraph:

```bash
mvn clean package -Pjanusgraph-release -Dgpg.skip=true -DskipTests=true
cd janusgraph-dist && mvn install -Pjanusgraph-docker -DskipTests=true docker:build
docker run -d -p 8182:8182 --name janusgraph janusgraph/server:latest
docker exec -i -t janusgraph /var/janusgraph/bin/gremlin.sh
```

**Comment by amcp** [comment]
One comment wasn't addressed. We should tell Gremlin Server to listen on 0.0.0.0. I think I've been burned by that in the past.

**Comment by sjudeng** [comment]
Issue resolved by #308
